features/apimgt/org.wso2.carbon.apimgt.publisher.feature/src/main/resources/publisher/source/src/app/components/Apis/Details/Environments/DeploymentOnbording.jsx

```diff
@@ -0,0 +1,312 @@
+import React, { useState } from 'react';
+import 'react-tagsinput/react-tagsinput.css';
```

Comment:
Why do we need tag input here?

Reply:
Fixed https://github.com/wso2/carbon-apimgt/pull/10270/commits/ee20a3b9b54f0df2bafd12c6a3e622938b22935e